microsoft/testsuites/core/timesync.py

```diff
@@ -129,14 +130,22 @@ def timesync_validate_ptp(self, node: Node) -> None:
     def timesync_check_unbind_clocksource(self, node: Node) -> None:
         # 1. Check clock source name is one of hyperv_clocksource_tsc_page,
         #  lis_hv_clocksource_tsc_page, hyperv_clocksource.
-        clocksource = [
+        clocksource_x86 = [
             "hyperv_clocksource_tsc_page",
             "lis_hyperv_clocksource_tsc_page",
             "hyperv_clocksource",
             "tsc",
         ]
+        clocksource_arm64 = [
+            "arch_sys_counter",
+        ]
         cat = node.tools[Cat]
         clock_source_result = cat.run(self.current_clocksource)
+        lscpu = node.tools[Lscpu]
+        if "x86_64" == lscpu.get_architecture():
```

Comment:
Add an enum class like below in os.py, and use it. 

```python
class Architecture(str, Enum):
    X64 = "x86_64"
    ARM64 = "aarch64"
```

Reply:
assign lscpu.get_architecture() to a value to avoid unnecessary method calls.

Reply:
Done